docs/articles/sde/introduction_to_sde.rst

```diff
@@ -17,6 +17,8 @@ Dependencies:
 * Vagrant
 * VirtualBox
 
+.. note:: Ubuntu 16.04 default repositories doesn't provide the latest version required
```

Comment:
If one has to add extra repositories for the SDE to work, please describe briefly the steps to get it working.

Reply:
You think is a good idea to flood with apt-get commands or links to the proper repositories this page? If so I will updated but, for my understanding, the page should easy to read, my note is just a warning that the previous procedure is not straight forward for Ubuntu users.

Reply:
Perhaps you can just mention what repos one has to add to get the latest version?

Reply:
I personally went to the website and download the debian package.

Reply:
Okay. I don't think I understand exactly the problem.

* Does it work in Ubuntu even with older versions of the components, or are newer versions required?
* If newer versions are required, can you update the PR with how to get them?

Reply:
Sorry if I don't explain well.
* if you use the default apt-get install vagrant/virtualbox you get older version that are not compatible with the git vagrant repository. You will get errors and freeze your PC sometimes. I don't know from which version will work properly, I just know that with the latest version available they works fine.
* To get the latest version I went to vagrant/virtualbox website and get the debian package to install them (instead of use the apt-get from a remote repos). Do you mean that I should add the links of the official vagrant/virtualbox distro?

Reply:
Yes, a link to the VBox and Vagrant sites, I guess linking to these two would suffice:

* https://www.virtualbox.org/wiki/Linux_Downloads (even mentions how to add to apt)
* https://www.vagrantup.com/downloads.html (only has raw .deb files it seems)